core/domain/acl_decorators.py

```diff
@@ -47,8 +46,7 @@ def test_can_play(self, exploration_id, **kwargs):
             exploration_id, strict=False)
 
         if rights_manager.check_can_access_activity(
-                self.user_id, self.actions, constants.ACTIVITY_TYPE_EXPLORATION,
-                exploration_rights):
+                self.user_id, self.actions, exploration_rights):
```

Comment:
Sounds better. For this, we will have to make rights_manager._get_activity_rights(...) to rights_manager.get_activity_rights(...). I should do it (exposing it to all modules), right?